pkg/sql/sem/builtins/all_builtins.go

```diff
@@ -56,9 +56,12 @@ func init() {
 		}
 	}
 
-	// Generate missing categories.
+	// Generate missing categories and volatilities.
 	for _, name := range AllBuiltinNames {
 		def := builtins[name]
+		if def.props.Volatility == 0 {
+			def.props.Volatility = tree.VolatilityImmutable
```

Comment:
Can you add something like: `TODO: instead of the default being immutable (which could be a problem if a new function forgets to set its volatility if not immutable), explicitly define volatility of all functions and add a test that asserts it is never the zero value.`